maxim/apis/maxim_apis.py

```
@@ -690,14 +690,14 @@ def get_folders(self) -> List[Folder]:
             raise Exception(e) from e
 
     def add_dataset_entries(
-        self, dataset_id: str, dataset_entries: List[DatasetEntry]
+        self, dataset_id: str, dataset_entries: List[Union[DatasetEntry, Dict[str, Any]]]
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)